doc/distrib/NodeHelpFiles/Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.ByTorusCoordinateSystemRadii.md

```diff
@@ -1,5 +1,6 @@
-## ByTorusCoordinateSystemRadii - Documentation
-This documentation file is auto generated by NodeDocumentationMarkdownGenerator, Version=2.14.0.3986, Culture=neutral, PublicKeyToken=null.
+## In-Depth
+In the example below, a T-Spline torus surface is created, with its origin at given Coordinate System `cs`. The minor and major radii of the shape are set by inputs `innerRadius` and `outerRadius`. The values for `innerRadiusSpans` and `outerRadiusSpans` control the definition of the surface along the two directions. The initial symmetry of the shape is specified by the `symmetry` input. If X or Y symmetry is set to True, the value of `outerRadiusSpans` must be a multiple of 4. Finally, the `inSmoothMode` input is used to switch between smooth and box mode preview of the T-Spline surface.
```

Comment:
It says that the `outerRadiusSpans` must be multiple of 4 of the X or Y symmetry is set to True, but there is no X and Y symmetry input, only Symmetry. Also, it seems the node works fine if the it isn't multiple of '4'. 
<img width="809" alt="image" src="https://github.com/DynamoDS/Dynamo/assets/5354594/03b88403-bb73-4b58-9b22-b524e4d6eec3">

Reply:
Ok this might be confusing. I'll replace the Radial symmetry with Axial and set it to a multiple of 4, and I'll rephrase this 

If **Axial** Symmetry is active along X or Y axes, the value of `outerRadiusSpans` must be a multiple of 4.

Reply:
If Axial symmetry applied to the shape is active for the X or Y axis, the value of `outerRadiusSpans` of the torus must be a multiple of 4. Radial symmetry has no such requirement.